gprofiler/utils/__init__.py

```diff
@@ -314,7 +314,7 @@ def pgrep_exe(match: str) -> List[Process]:
 def pgrep_maps(match: str) -> List[Process]:
     # this is much faster than iterating over processes' maps with psutil.
     result = run_process(
-        f"grep -lP '{match}' /proc/*/maps",
+        f"grep -lE '{match}' /proc/*/maps",
```

Comment:
You can comment here explaining about the choice of `-E` over `-P` (to support systems where grep is not PCRE)